sonar-core/src/test/java/org/sonar/core/config/CorePropertyDefinitionsTest.java

```diff
@@ -31,4 +37,25 @@ public void all() {
     List<PropertyDefinition> defs = CorePropertyDefinitions.all();
     assertThat(defs.size()).isGreaterThan(9);
   }
+
+  @Test
+  public void jdbc_password_property_has_password_type() {
+    List<PropertyDefinition> defs = CorePropertyDefinitions.all();
+    Optional<PropertyDefinition> prop = from(defs).filter(new HasKeyPredicate(DatabaseProperties.PROP_PASSWORD)).first();
+    assertThat(prop.isPresent()).isTrue();
```

Comment:
since `prop.get` will fail anyway at the next line, this assertion is not necessary